[WIP] Add Bazarr for subtitles management

marcomarinho opened the pull request.

marcomarinho committed:
Add Bazarr for subtitles management

JaviPege commented:
Just reviewed it, looks fine, this week I’ll be testing it on a clean install. Thanks again Marco!

marcomarinho committed:
Remove service user UIDs
let's keep it simple , use the same, won't harm and it gets problem with folder perms

marcomarinho committed:
autoscan: copy file to config folder

marcomarinho committed:
Remove setup.sh.original and fix homarr folder creation

marcomarinho committed:
Move from Overseerr to Seerr
use develop tag for now

marcomarinho committed:
plex improvements

marcomarinho committed:
fix user perms
